lang/cs/Org.Apache.REEF.Client/API/IREEFClient.cs

```diff
@@ -43,5 +45,13 @@ public interface IREEFClient
         /// <returns>IDriverHttpEndpoint</returns>
         [Unstable("0.13", "Working in progress for what to return after submit")]
         IDriverHttpEndpoint SubmitAndGetDriverUrl(IJobSubmission jobSubmission);
+
+        /// <summary>
+        /// Returns the application status in running the job
+        /// </summary>
+        /// <param name="appId"></param>
+        /// <returns></returns>
+        [Unstable("0.14", "Working in progress for rest API status returned")]
+        Task<State> GetJobFinalStatus(string appId);
```

Comment:
How would one obtain the `appId`? 

I still wonder why we can't just return the app id as part of `SubmitAndGetDriverUrl`?